### PR TITLE
DOCS: Updating links and images

### PR DESCRIPTION
**Pull request by syedjawadakhtar**
Major changes to the doc includes: Adding links and images

### CODE REVIEW

**Comment by pnijhara** [comment]
@syedjawadakhtar thank you for the patch. Let me review it first.

File: docs/computer/Acoustics.md [hunk 1]
```diff
@@ -4,67 +4,91 @@ author: Syed Jawad Akhtar
 email: syedjawadakhtar@gmail.com
 ---
 
-> :warning: **Documentation is still in progress** 
+> :warning: **WIP**
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: Please remove emojis.

File: docs/computer/Acoustics.md [hunk 2]
```diff
@@ -4,67 +4,91 @@ author: Syed Jawad Akhtar
...
+#### Introduction
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: Sub-headings should go in level.
Try to give only one h1 followed by sub-headings as h2. If required then give h3 and h4.

File: docs/computer/Acoustics.md [hunk 3]
```diff
@@ -4,67 +4,91 @@ author: Syed Jawad Akhtar
...
 
 An acoustic localization system detects a frequency emitting source at a distance, using Time Difference of Arrival (TDOA) of the sound wave.
 
-#### Our Goal
-We chose this task to be ocmpleted as it's very tedious to make this system incorporate with the movements of vehicle as well as the equipments used is comparitively costly and complicated. And because of these downsides for student teams the points to accomplish has been increased by a large amount as compared to performing other tasks. Thus helping the teams win with large margins. Check [SAUVC Rulebook](https://sauvc.org/rulebook/) and [mission and scoring](https://robosub.org/resources/) of Robosub competitions.
+#### Goal
+The types of equipment used are comparatively costly and complicated to make this system fuse with the operations of the vehicle. For this reason, the points to accomplish the tasks including a pinger is higher as compared to other tasks. 
+Check [SAUVC Rulebook](https://sauvc.org/rulebook/) and [mission and scoring](https://robosub.org/resources/) of Robosub competitions.
 
 #### Objective
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: Same! h4 does not required. Can go with h2.

File: electronics/Acoustics.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+---
+title: Initial Acoustics Implementation
+author: Syed Jawad Akhtar
+email: syedjawadakhtar@gmail.com
+---
+
+> :warning: **Documentation is still in progress**
```
Comment (pnijhara) [inline electronics/Acoustics.md:7]: Remove emojis

File: electronics/Acoustics.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+title: Initial Acoustics Implementation
+author: Syed Jawad Akhtar
+email: syedjawadakhtar@gmail.com
+---
+
+> :warning: **Documentation is still in progress** 
+
+## Acoustics Implementation - Survey, procurement & implementation
+
+#### Introduction about acoustics and competitions using it
```
Comment (pnijhara) [inline electronics/Acoustics.md:11]: Can go as h3. Or take a look again if they follow levels or not.

File: electronics/BMS_for_lithium_polymer_battery_pack.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+## Our approach:
+To design a threefold circuit:
+- over charge protection
+- discharge protection
+- battery level indication
+
+We decided to go the analogue rough for the BMS because we wanted to design a PCB and for current and temperature sensing we used an arduino simply because it is easy to logging data, we used lm 35 for temperature and acs712 for current sensing. In case the temperature exceeds the battery's manufacture specified operating temperature the battery will be disconnected from the output power.
+
+
```
Comment (pnijhara) [inline electronics/BMS_for_lithium_polymer_battery_pack.md:20]: Remove empty lines.

Comment (syedjawadakhtar) [reply electronics/BMS_for_lithium_polymer_battery_pack.md:20]: Removed. And done other changes as well. Please check.

File: electronics/Acoustics.md (hunk 4 above)
Comment (pnijhara) [reply electronics/Acoustics.md:7]: Please remove this warning too!

**Pull request merged**